### DISCUSSION TITLE
Pendaftaran Peserta

### DISCUSSION THREAD

**Discussion by rahmanramsi** [OpenSynergic/leconfe#144] [Ideas]
Role admin
- System bisa membuat Registration Fees yang akan diterima oleh conference.
- System bisa melakukan setting untuk system pembayaran secara manual 
atau otomatis dari provider pihak ketiga, seperti paypal, xendit, paddle, dll.
- Admin bisa melihat list peserta yang sudah mendaftar, 
beserta informasi lain seperti status pembayaran, dll.
Admin bisa melakukan aksi untuk merubah status pembayaran 
peserta ketika setting pembayaran dilakukan secara manual.
![image](https://github.com/OpenSynergic/leconfe/assets/48276043/d8b25c72-8590-4c25-b07a-00c47fe62908)

Role Peserta
- Peserta bisa mendaftar sebagai category yang sudah ditentukan oleh conference. 
Contoh : General Presenter, Non-Presenter, dll.
- Peserta harus melakukan pembayaran setelah melakukan pendaftaran sebagai peserta.
- Peserta bisa melihat informasi pendaftaran yang sudah dilakukan

![image](https://github.com/OpenSynergic/leconfe/assets/48276043/2441bb63-98fb-4336-b2a1-744e7e3503a2)

**Comment by rahmanramsi** [comment]
Pertanyaan :
Ketika melakukan pendaftaran peserta, apakah setiap kategori punya syarat" tertentu yg tidak ada dikategori lain? Semisal ketika melakukan pendaftaran sebagai Presenter, apakah ada data yg perlu dilengkapi terlebih dahulu sebelum pendaftaran itu bisa diterima?


@gokilalma

**Comment by navotera** [comment]
saat ini pembayaran hanya transfer bank saja dengan tambahan note
